Template/README.md

```diff
@@ -1 +1,18 @@
-# {PROJECT}
\ No newline at end of file
+# {PROJECT}
+
+### Installation
+
+**CocoaPods:**
+Add the line `pod "{PRODUCT}"` to your `Podfile`
+
+**Carthage:**
+
+Add the line `github "yourusername/{PRODUCT}"` to your `Cartfile`
+
+**Manual:**
+
+Clone the repo and drag the file `{PRODUCT}.swift` into your Xcode project.
+
+**Swift Package Manager:**
+
+Add the line `.Package(url: "https://github.com/yourusername/{PRODUCT}.git", majorVersion: 2)` to your `Package.swift` file.
```

Comment:
We should use the GitHub URL that the user gives here instead 😄 (with `.git` appended) Also, how about we set the `majorVersion` to `2` by default?